util/subprocesses.py

```diff
@@ -519,12 +517,14 @@ def shellify(cmd):
 
 
 def timeSubprocess(command, ignoreStderr=False, combineStderr=False, ignoreExitCode=False,
-                   cwd=os.getcwdu(), env=os.environ, vb=False):
+                   cwd=None, env=None, vb=False):
     """
     Calculate how long a captureStdout command takes and prints it.
 
     Return the stdout and return value that captureStdout passes on.
     """
+    env = env or os.environ
+    cwd = cwd or (os.getcwd().decode("utf-8", errors="replace") if sys.version_info.major == 2 else os.getcwd())
```

Comment:
This was fixed in #123, it appeared here due to rebasing, it will go away after this PR is updated.